Header header
int32[] ranges_top
int32[] ranges_bottom
float32 angle_span = 90
int32 n_rois=21
int32 frame_height=480
int32 frame_width=640
